linux_os/guide/system/software/updating/dnf-automatic_apply_updates/oval/shared.xml

```diff
@@ -0,0 +1,14 @@
+{{{
+oval_check_config_file(
+    path="/etc/dnf/automatic.conf",
+    prefix_regex="^[\s]*\[commands](?:[^\\n\[]*\\n+)+?[\s]*",
+    parameter="apply_updates",
+    value="yes",
+    separator_regex="[\s]*=[\s]*",
+    missing_parameter_pass=false,
+    application="dnf-automatic",
+    multi_value=false,
+    missing_config_file_fail=true,
+    supported_platforms=["rhel8","ol8","fedora"]
```

Comment:
Please remove this `supported_platforms` option. 

As per: https://github.com/ComplianceAsCode/content/pull/4596/files it will reuse the `prodtypes` value from rule.yml to build the `<affected>` xml tag.

Reply:
@ggbecker I will remove it